### ISSUE TITLE
Shaders do not work

### ISSUE THREAD

**Issue by ninryu** [LunaPixelStudios/Better-Minecraft#108]
### Describe the Bug

despite installing Optifine and and shaders, they do not work or show up at the menu. This issue only occurs in the normal version of Better Minecraft, as in Better Minecraft Plus the shaders work fine.

### Reproduction Steps

1.
2.
3.
...

### Operating System

Windows10

### Minecraft Version

1.18

### Modloader

Forge

### Modpack Version

V16.5

### Optifine

I used Optifine

**Comment by SharkieTV** [comment]
We dont deal with Optifine issues and it does work for me but just send the latest.log and ill check it out.

**Comment by switchnlade** [comment]
Hmm, why are these missing? How did you install your modpack? These four mods should be included as they are on my end.
```
[17:25:53] [Render thread/ERROR]: This world was saved with mod corpse which appears to be missing, things may not work well
[17:25:53] [Render thread/ERROR]: This world was saved with mod dungeons_arise which appears to be missing, things may not work well
[17:25:53] [Render thread/ERROR]: This world was saved with mod backpacked which appears to be missing, things may not work well
[17:25:53] [Render thread/ERROR]: This world was saved with mod xpbook which appears to be missing, things may not work well
```

**Comment by SharkieTV** [comment]
> [latest.zip](https://github.com/LunaPixelStudios/Better-Minecraft/files/8054112/latest.zip)

This zip is for Fabric version not forge? So why are you saying its Forge when its not, why are you trying to add Optifine to Fabric when it already has Iris?
Also why aren't you mentioning you added mods in that zip which also means that's probably what's causing our crash in general.

**Comment by switchnlade** [comment]
The log included is for forge: "forge-39.0.75, D:\Overwolf\Instances\Better Minecraft [FORGE] - 1.18.1"
Overall, the instance did not install properly and was modified as you said.

**Comment by SharkieTV** [comment]
No reponse
